c2pa/claim-templates/claim_create.json

```diff
@@ -0,0 +1,88 @@
+{
+    "vendor": "starlinglab",
+    "recorder": "{{recorder}}",
+    "assertions": [
+        {
+            "label": "stds.schema-org.CreativeWork",
+            "data": {
+                "@context": "https://schema.org",
+                "@type": "CreativeWork",
+                "author": [
+                    {
+                        "@type": "{{author_type}}",
+                        "credential": [
+                        ],
+                        "identifier": "{{author_identifier}}",
+                        "name": "{{author_name}}"
+                    }
+                ]
+            }
+        },
+        {
+            "label": "c2pa.actions",
+            "data": {
+                "actions": [
+                    {
+                        "action": "c2pa.created",
+                        "when": "{{authenticated_asset_created_timestamp}}"
```

Comment:
@anaulin this should match `starling:assetCreatedTimestamp` and is taken from POST request's `meta` -> `\"timestamp\":1635627729773`.